packages/xo-web/src/common/store/actions.js

```diff
@@ -58,3 +58,15 @@ export const setHomeVmIdsSelection = createAction(
   'SET_HOME_VM_IDS_SELECTION',
   homeVmIdsSelection => homeVmIdsSelection
 )
+export const setHubInstallLoadingState = createAction(
+  'SET_HUB_INSTALL_LOADING_STATE',
+  hubInstallLoadingState => hubInstallLoadingState
+)
+export const markHubResourceAsInstalling = createAction(
+  'MARK_HUB_RESOURCE_AS_INSTALLING',
+  id => ({ [id]: true })
```

Comment:
Keep things simple:
```suggestion
  id => id
```

And handle it properly in the reducer.